test/unit/webdriveragent-specs.js

```diff
@@ -16,7 +16,7 @@ let fakeConstructorArgs = {
 
 describe('Constructor', () => {
   it('should have a binary for the webdriver agent', async () => {
-    let agent = new WebDriverAgent(fakeConstructorArgs);
+    let agent = new WebDriverAgent({}, fakeConstructorArgs);
     await fs.exists(agent.agentPath);
```

Comment:
I know this line was here before, but does our `fs.exists` return a boolean, or throw if not exists? If the former then we should assert something here